libs/payments/customer/src/lib/util/stripeInvoiceToFirstInvoicePreviewDTO.ts

```diff
@@ -24,9 +24,19 @@ export function stripeInvoiceToInvoicePreviewDTO(
       )
     : null;
 
+  const unusedAmountTotal = invoice.lines.data
+    .filter((line) => line.amount < 0)
+    .reduce((sum, line) => sum + line.amount, 0);
```

Comment:
`invoice.lines` can contain a bunch of different things, and IMO it's risky to sum the amounts together on an invoice to get the unusedAmountTotal. 

If the intention is to calculate the total proration, maybe consider making use of the `invoice.lines.data.proration` flag.